SmartDeviceLink-iOS.podspec

```diff
@@ -6,462 +6,15 @@ s.summary      = "Connect your app with cars!"
-'SmartDeviceLink/SDLSystemCapabilityType.h',
-'SmartDeviceLink/SDLSystemContext.h',
-'SmartDeviceLink/SDLSystemRequest.h',
-'SmartDeviceLink/SDLTBTState.h',
-'SmartDeviceLink/SDLTemperature.h',
-'SmartDeviceLink/SDLTemperatureUnit.h',
-'SmartDeviceLink/SDLTemplateConfiguration.h',
-'SmartDeviceLink/SDLTemplateColorScheme.h',
-'SmartDeviceLink/SDLTextAlignment.h',
-'SmartDeviceLink/SDLTextField.h',
-'SmartDeviceLink/SDLTextFieldName.h',
-'SmartDeviceLink/SDLTimerMode.h',
-'SmartDeviceLink/SDLTireStatus.h',
-'SmartDeviceLink/SDLTouch.h',
-'SmartDeviceLink/SDLTouchCoord.h',
-'SmartDeviceLink/SDLTouchEvent.h',
-'SmartDeviceLink/SDLTouchEventCapabilities.h',
-'SmartDeviceLink/SDLTouchManager.h',
-'SmartDeviceLink/SDLTouchManagerDelegate.h',
-'SmartDeviceLink/SDLTouchType.h',
-'SmartDeviceLink/SDLTPMS.h',
-'SmartDeviceLink/SDLTransmissionType.h',
-'SmartDeviceLink/SDLTriggerSource.h',
-'SmartDeviceLink/SDLTTSChunk.h',
-'SmartDeviceLink/SDLTurn.h',
-'SmartDeviceLink/SDLTurnSignal.h',
-'SmartDeviceLink/SDLUnpublishAppService.h',
-'SmartDeviceLink/SDLUnpublishAppServiceResponse.h',
-'SmartDeviceLink/SDLUnregisterAppInterface.h',
-'SmartDeviceLink/SDLUnregisterAppInterfaceResponse.h',
-'SmartDeviceLink/SDLUnsubscribeButton.h',
-'SmartDeviceLink/SDLUnsubscribeButtonResponse.h',
-'SmartDeviceLink/SDLUnsubscribeVehicleData.h',
-'SmartDeviceLink/SDLUnsubscribeVehicleDataResponse.h',
-'SmartDeviceLink/SDLUnsubscribeWaypoints.h',
-'SmartDeviceLink/SDLUnsubscribeWaypointsResponse.h',
-'SmartDeviceLink/SDLUpdateMode.h',
-'SmartDeviceLink/SDLUpdateTurnList.h',
-'SmartDeviceLink/SDLUpdateTurnListResponse.h',
-'SmartDeviceLink/SDLVehicleDataActiveStatus.h',
-'SmartDeviceLink/SDLVehicleDataEventStatus.h',
-'SmartDeviceLink/SDLVehicleDataNotificationStatus.h',
-'SmartDeviceLink/SDLVehicleDataResult.h',
-'SmartDeviceLink/SDLVehicleDataResultCode.h',
-'SmartDeviceLink/SDLVehicleDataStatus.h',
-'SmartDeviceLink/SDLVehicleDataType.h',
-'SmartDeviceLink/SDLVentilationMode.h',
-'SmartDeviceLink/SDLVehicleType.h',
-'SmartDeviceLink/SDLVersion.h',
-'SmartDeviceLink/SDLVideoStreamingCapability.h',
-'SmartDeviceLink/SDLVideoStreamingCodec.h',
-'SmartDeviceLink/SDLVideoStreamingFormat.h',
-'SmartDeviceLink/SDLVideoStreamingProtocol.h',
-'SmartDeviceLink/SDLVideoStreamingState.h',
-'SmartDeviceLink/SDLVoiceCommand.h',
-'SmartDeviceLink/SDLVrCapabilities.h',
-'SmartDeviceLink/SDLVrHelpItem.h',
-'SmartDeviceLink/SDLWarningLightStatus.h',
-'SmartDeviceLink/SDLWayPointType.h',
-'SmartDeviceLink/SDLWeatherAlert.h',
-'SmartDeviceLink/SDLWeatherData.h',
-'SmartDeviceLink/SDLWeatherServiceData.h',
-'SmartDeviceLink/SDLWeatherServiceManifest.h',
-'SmartDeviceLink/SDLWiperStatus.h',
-'SmartDeviceLink/SDLWindowCapability.h',
-'SmartDeviceLink/SDLWindowState.h',
-'SmartDeviceLink/SDLWindowStatus.h',
-'SmartDeviceLink/SDLWindowType.h',
-'SmartDeviceLink/SDLWindowTypeCapabilities.h',
-'SmartDeviceLink/SmartDeviceLink.h',
-]
+s.public_header_files = 'SmartDeviceLink/public/*.h'
```

Comment:
The library is not compiling when I install via CocoaPods due to **SDLPermissionManager.h** being in the **private** folder when it should be in the public folder.